src/java/org/apache/cassandra/service/StorageServiceMBean.java

```diff
@@ -769,9 +769,22 @@ default int upgradeSSTables(String keyspaceName, boolean excludeCurrentVersion,
      * @param keyspace Name of the keyspace which to rebuild or null to rebuild all keyspaces.
      * @param tokens Range of tokens to rebuild or null to rebuild all token ranges. In the format of:
      *               "(start_token_1,end_token_1],(start_token_2,end_token_2],...(start_token_n,end_token_n]"
+     * @param specificSources list of sources that can be used for rebuilding. Mostly other nodes in the cluster.
```

Comment:
nit: `list of sources that can be used for rebuilding. Must be other nodes in the cluster. The format of the string is comma separated values.`